enclave-tls/rules/sgx_rules.mk

```diff
@@ -102,7 +102,7 @@ ifeq ($(Tls_Wolfssl),1)
   Enclave_Cflags += $(Wolfssl_Sgx_Cflags)
 endif
 $(App_Name)_t.o: $(App_Name)_t.c
-	$(CC) $(Enclave_Cflags) -c $< -o $@
+	$(CC) $(Enclave_Cflags) -DHAVE_TM_TYPE -c $< -o $@
```

Comment:
Plz add the comment about this macro. In addition, is it possible to add to Enclave_Cflags? Furthermore, you last pr defines this macro for wolfssl_env.mk but now it is removed. Why?

Reply:
it can't be placed into Enclave_Cflags, otherwise the re-definition will promoted. Remove HAVE_TM_TYPE from wolfssl_env.mk is because it will cause ./src/external/wolfssl/wolfssl/IDE/LINUX-SGX/libwolfssl.sgx.static.lib.a compiling error.
I will give a comment here.